HostIOInterceptor.cs

```diff
@@ -127,9 +152,9 @@ public void DetachFromHost()
                                                             string message,
                                                             Collection<FieldDescription> descriptions)
         {
-            if (externalUI == null)
+            if (externalUI != null)
```

Comment:
Looks like a little too much find/replace happened.  Several of these should still be `== null` :)

Reply:
Oh good grief. Good catch. The silly thing is, I pasted each one of those individually and customized the thrown messages 😝